### ISSUE TITLE
DashboardAppBar header elements are not keyboard operable.

### ISSUE THREAD

**Issue by jennlove-um** [tl-its-umich-edu/my-learning-analytics#1150]
### Expected behavior (A description of what you expected to happen) : 
When using the keyboard (Tab keyboard button) to navigate through links and options on the DashboardAppBar component, users should be able to tell visually what has focus and be able to select all options using Enter keyboard button.

This was originally resolved with #803 

### Describe the bug (Tell us what happens instead of the expected behavior) :
Accessibility testing found that the header elements are not keyboard operable. Cannot tab to them. The two links are "My Learning Analytics:" and "GERMAN 102 WN 2020" - they have a tabindex="-1" is why. 

### Steps to Reproduce :
1.  Go to MyLA for a course
2. Tab through the page - the My Learning Analytics link and the course name link cannot be selected by tabbing

**Comment by jonespm** [comment]
It looks like this was done as part of #807 (rather than #803) in the last release. I remember mentioning this tabindex in the review but not going back to revisit it.

**Comment by jennlove-um** [comment]
Testing passes: I was able to tab to the headers and select them.
